apps/client/src/routes/index.tsx

```diff
@@ -1,36 +1,36 @@
-import { createFileRoute } from '@tanstack/react-router'
-import logo from '../logo.svg'
-import '../App.css'
+import React from 'react';
+import { createFileRoute } from '@tanstack/react-router';
+import { Box } from '@mui/material';
+import Navbar from '../components/navbar';
+import Footer from '../components/footer';
+import HeroBanner from '../components/herobanner';
+import '../App.css';
```

Comment:
Could we use the @/ instead of relative path for our libraries? A bit more clearer